src/services/templating.ts

```diff
@@ -11,8 +11,10 @@ export async function evaluate(templateId: string, context: object) {
     throw new Error(`Template with id "${templateId}" not found.`)
   }
   switch (templateEl.getAttribute("type")) {
+    case "text/x-liquid-template":
```

Comment:
Can you please share link to the source that defines this type? I couldn't find it in the official list https://www.iana.org/assignments/media-types/media-types.xhtml